pycm/pycm_obj.py

```diff
@@ -220,12 +243,25 @@ def save_stat(
             if summary:
                 class_list = SUMMARY_CLASS
                 overall_list = SUMMARY_OVERALL
+            classes = self.classes
+            table = self.table
             file = open(name + ".pycm", "w", encoding="utf-8")
-            matrix = "Matrix : \n\n" + table_print(self.classes,
-                                                   self.table) + "\n\n"
-            normalized_matrix = "Normalized Matrix : \n\n" + \
-                                table_print(self.classes,
-                                            self.normalized_table) + "\n\n"
+            if sparse is True:
+                if self.sparse_matrix is None:
+                    self.sparse_matrix = sparse_matrix_calc(classes, table)
+                matrix = "Matrix : \n\n" + \
+                    sparse_table_print(self.sparse_matrix) + "\n\n"
+                if self.sparse_normalized_matrix is None:
+                    table = normalized_table_calc(classes, table)
```

Comment:
Why recalculation?
We have `self.normalized_table`

Reply:
I hadn't realized `self.normalized_table` as an attribute :smile: